### PR TITLE
fix(docs): removed duplicate line

### PR DESCRIPTION
**Pull request by httpsOmkar**

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @httpsOmkar! I've created [OSPR-4078](https://openedx.atlassian.net/browse/OSPR-4078) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://open.edx.org/wp-content/uploads/2019/01/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

File: src/auth/index.js [hunk 1]
```diff
@@ -20,7 +20,6 @@
  *   BASE_URL,
  *   LMS_BASE_URL,
  *   LOGIN_URL,
- *   LOGIN_URL,
```
Comment (davidjoy) [inline src/auth/index.js]: Ah, this is supposed to be `LOGOUT_URL`, along with line 34 below...

Comment (davidjoy) [reply src/auth/index.js]: Line 34 should read:
```
 *   logoutUrl: LOGOUT_URL,
```

Comment (httpsOmkar) [reply src/auth/index.js]: Fixed

**Comment by edx-webhook** [comment]
@httpsOmkar 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/frontend-platform&entry.867055334=https://github.com/edx/frontend-platform/pull/57&entry.1484655318=https://github.com/httpsOmkar&entry.752974735=2020-02-04+08:31&entry.1917517419=2020-02-04+17:21&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by natabene** [comment]
@httpsOmkar  Thank you for the contribution! Can you please send in a signed Contributor Agreement mentioned in the earlier comment in the next 2 days? You only need to do this once and we can then review all your future code contributions. Without it, unfortunately, we will need to revert this PR. Sorry for confusion. Thanks

**Comment by httpsOmkar** [comment]
@natabene I just sent an copy of my agreement to the legal address.
Thanks

**Comment by natabene** [comment]
@httpsOmkar Thank you, I will wait for our legal team to confirm and will post here once you are all set.

**Comment by natabene** [comment]
@httpsOmkar 
Thank you for joining the Open edX community! We have added your user to our records and can now review your code going forward.

**Pull request merged**